### DISCUSSION TITLE
How to enable showing colors of indent on current context (line) only in V3?

### DISCUSSION THREAD

**Discussion by ecosse3** [lukas-reineke/indent-blankline.nvim#641] [Q&A] [answered]
Hi,

I started using V3 version today, and I cannot make showing colors of indent on current context line only in V3.
In V2 it was working by providing option:

```lua
  show_current_context = true,
  show_current_context_start = false,
```

Now I cannot find it. My current indents always have colors, which is a bit distracting:
![image](https://github.com/lukas-reineke/indent-blankline.nvim/assets/17691791/d8476f1e-7ebf-44d4-a327-e6b2f7b7e3b4)

I want to have `grey` indents when they are not in current context.

**Comment by lukas-reineke** [comment]
Set `indent.highlight` to a grey highlight group

**Reply by ecosse3** [reply]
That's set all indents to grey color, but I want old behaviour that current context indent will be highlighted with a colorful color:

![image](https://github.com/lukas-reineke/indent-blankline.nvim/assets/17691791/f7905964-01f6-40d3-8572-4c7b5b00c8d8)

**Comment by tigion** [comment, marked as answer]
I think there is a misunderstanding:
- `v3` scope idention level != `v2` current idention level (`show_current_context`)

**Reply by ecosse3** [reply]
Yes, I get it. But is there an similar option to `show_current_context` in `v3`?

**Reply by lukas-reineke** [reply]
> is there an similar option to show_current_context in v3

There is not. Scope is the scope of the program, not the indentation level. It is explained in detail in `:help ibl.config.scope`.
You can add other node types to the scope manually with `:help ibl.config.scope.include` to get the behavior from v2 if you prefer.
